packages/zosmf/src/ZosmfBaseHandler.ts

```diff
@@ -40,11 +40,13 @@ export abstract class ZosmfBaseHandler implements ICommandHandler {
 
     /**
      * Loaded z/OSMF profile if needed
+     * @deprecated
      */
     protected mZosmfProfile: IProfile;
 
     /**
      * Loaded z/OSMF profile with meta information
+     * @deprecated
      */
     protected mZosmfLoadedProfile: IProfileLoaded;
```

Comment:
Are this newly deprecated properties?

I believe they still exist in the next branch 😋 

Should we create an issue to remove them/replace them over there?

Reply:
I only deprecated them because the code around them didn't actually do anything, and the process of setting them used deprecated functions.

Reply:
After in person discussion, we are planning to remove these in V3 since nothing sets the protected variables anymore.